sdk/tests/conformance2/context/methods-2.html

```diff
@@ -247,11 +249,7 @@
 "isSampler",
 "bindSampler",
 "samplerParameteri",
-"samplerParameteriv",
-"samplerParameteriv",
 "samplerParameterf",
-"samplerParameterfv",
-"samplerParameterfv",
```

Comment:
Fair point. I'd not double-checked the spec and realized these had already been removed there some time ago.